source/_integrations/hunterdouglas_powerview.markdown

```diff
@@ -203,6 +206,15 @@ Gen 1 and Gen 2 Only.
 </div>
 Set the type for connected power source. Available options are Hardwired Power Supply, Battery Wand and Rechargeable Battery
 
+## Number Entities
```

Comment:
```suggestion
## Number entities
```
I realize this page uses some heading style capitalization. But in general, we use sentence-style capitalization.